src/app/services/api.ts

```diff
@@ -630,6 +630,41 @@ export class ApiService {
     return this.http.get<Document[]>(`${this.apiPath}/${queryString}`, {});
   }
 
+  /**
+   * Get all documents.
```

Comment:
could be a little more descriptive... get all documents for what?

Reply:
I do agree that we should write descriptive doc blocks but in this case, I'm not sure how. This call is basically the same as the PMO app's getAll Contracts call: https://github.com/bcgov/gdx-agreements-tracker/blob/development/backend/src/controllers/contacts.js#L29

This is a common data type in the app and we just need to get all of them.